UPGRADE-6.3.md

```diff
@@ -155,6 +155,23 @@ done by adding the following line after the module list definition:
 There are additional helper methods to simplify adding typical backend modules, but it is optional to use these methods.
 See `src/CoreBundle/private/library/classes/pagedefFunctions.inc.php` for reference.
 
+## Main Menu Changes
+
+This release of Chameleon System features a new main menu that is displayed as a sidebar while the old main menu (now
+called "classic main menu" is deprecated and will be removed in a future release. The new menu was restructured and some
+menu items were renamed to improve comprehensibility of the menu structure.
+
+The content boxes of the classic main menu are unchanged to give users time to get accustomed to the new menu. It
+didn't make sense to keep old menu item names, though, so be aware that some menu items were changed. The changes
+should be quite straightforward.
+
+Also, some menu items that were located in the top bar were now moved to the sidebar. Finally, the backend modules that
+were called in a popup window, like navigation, product search index generation and sanity check, now open inline.
+
+From a technical point no changes are required if the project is a shop system. Projects that are pure CMS systems
+should consider removing menu categories that are nevertheless created during migration. Note that there will be some
+update messages that complain about missing shop tables and modules in this case, which can be ignored. 
```

Comment:
Error messages?

Reply:
Changed to "error messages during migration"